seqr/views/utils/permissions_utils.py

```diff
@@ -37,33 +63,64 @@ def check_project_permissions(project, user, **kwargs):
         user=user, project=project))
 
 
-def check_user_created_object_permissions(obj, user):
-    if user.is_staff or obj.created_by == user:
+def check_user_created_object_permissions(obj, user, session=None):
+    if is_staff(user, session) or obj.created_by == user:
         return
     raise PermissionDenied("{user} does not have edit permissions for {object}".format(user=user, object=obj))
 
 
-def check_multi_project_permissions(obj, user):
+def check_multi_project_permissions(obj, user, session=None):
     for project in obj.projects.all():
         try:
-            check_project_permissions(project, user)
+            check_project_permissions(project, user, session)
```

Comment:
Codacy found an issue: [Too many positional arguments for function call](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6211929)